bugsnag/utils.py

```diff
@@ -1,78 +1,92 @@
 from __future__ import division, print_function, absolute_import
 
 import inspect
+import json
 import traceback
 import six
 
 import bugsnag
 
-try:
-    import json
-except ImportError:
-    import simplejson as json
-
 
 MAX_PAYLOAD_LENGTH = 128 * 1024
 MAX_STRING_LENGTH = 1024
 
 
-def sanitize_object(obj, **kwargs):
-    filters = kwargs.get("filters", [])
-
-    if isinstance(obj, dict):
-        clean_dict = {}
-        for k, v in six.iteritems(obj):
-            # Remove values for keys matching filters
-            if any(f.lower() in k.lower() for f in filters):
-                clean_dict[k] = "[FILTERED]"
-            else:
-                clean_obj = sanitize_object(v, **kwargs)
-                if clean_obj is not None:
-                    clean_dict[k] = clean_obj
-
-        return clean_dict
-    elif any(isinstance(obj, t) for t in (list, set, tuple)):
-        return [sanitize_object(x, **kwargs) for x in obj]
-    elif any(isinstance(obj, t) for t in (bool, float, int)):
-        return obj
-    else:
+class JSONEncoder(json.JSONEncoder):
```

Comment:
If these 2 classes were `import`ed into the same namespace they would caused a conflict unless they were imported with `as` having different names. I would call this `SanitizingJSONEncoder`

I'm being nitpicky so feel free to ignore 😄